packages/ui/src/components/va-data-table/VaDataTable-filter.demo.vue

```diff
@@ -0,0 +1,244 @@
+<template>
```

Comment:
Can we make this demo shorter?

These are probably not needed:
* checkboxes
* debounce
* alert
* custom filtering function

Demo ideally should be as simple as possible.